mlir/lib/Transforms/RemoveDeadValues.cpp

```diff
@@ -742,7 +742,48 @@ static void processBranchOp(BranchOpInterface branchOp, RunLivenessAnalysis &la,
 static void cleanUpDeadVals(RDVFinalCleanupList &list) {
   LDBG() << "Starting cleanup of dead values...";
 
-  // 1. Operations
+  // 1. Blocks
```

Comment:
Can you add a comment that explains the cleanup steps must happen in this order?